ansible/ci-playbooks/roles/ci-base-setup/tasks/redhat.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: ci-base-setup (redhat) | Set hostname
+  hostname:
+    name: "{{ inventory_hostname }}"
```

Comment:
@rleigh-codelibre - why are you forcing a hostname here? What if I'm running this on a system with an already-assigned hostname, and use, say, a cname in my inventory? e.g. octopus/eel, trout, etc?